package.json

```diff
@@ -51,7 +51,7 @@
     "wd": "^0.4.0"
   },
   "devDependencies": {
-    "jasmine-node": "~1",
-    "jshint": "^2.6.0"
+    "jshint": "^2.6.0",
+    "appium": "1.5.3"
```

Comment:
We call this method only when running Appium tests locally, so we save time in all other runs - we don't have to install pretty heavy `appium` module when running on Sauce Labs, for example.